src/main/resources/data-repository-openapi.yaml

```diff
@@ -268,6 +278,10 @@ paths:
             $ref: '#/definitions/JobModel'
         303:
           description: Redirect for object successful creation
+          headers:
```

Comment:
I think @thathert wanted the 200 response to also have the location header so we are always telling the client where to go to poll. Could add this after merge if you like, but probably simple to do now.